modules/browsiRtdProvider.js

```diff
@@ -332,13 +335,23 @@ export const browsiSubmodule = {
   getBidRequestData: setBidRequestsData
 };
 
-function getTargetingData(uc) {
+function getTargetingData(uc, c, us, a) {
   const targetingData = getRTD(uc);
+  const auctionId = a.auctionId
   uc.forEach(auc => {
     if (isNumber(_ic[auc])) {
       _ic[auc] = _ic[auc] + 1;
     }
+    const transactionId = a.adUnits.find(adUnit => adUnit.code === auc).transactionId;
```

Comment:
I believe that in general there may be more than one adUnit, and more than one transactionId, for a given code. Does it makes sense here to pick the first?

Reply:
Shouldn't ad unit code be unique?
This is how it's documented - 

> Unique identifier that you create and assign to this ad unit. Used to set query string targeting on the ad. If using GPT, we recommend setting this to slot element ID.
https://docs.prebid.org/dev-docs/publisher-api-reference/addAdUnits.html

Reply:
The documentation could be improved; it also says: 

https://docs.prebid.org/dev-docs/adunit-reference.html#twin-adunit-codes
> It’s ok to have multiple AdUnits with the same code. This can be useful in scenarios where bidders have different capabilities for the same spot on the page. 

If your aim is to emit an event for each adUnit, I'd just loop through them directly instead of the codes.

Reply:
I believe the use case for them not being unique still winds up targeting the same slot on the page.

So, perhaps the browsi use case does in fact only want to emit for each adUnitCode?

I'll let the browsi team determine that @omerBrowsi

Reply:
Correct. adUnitCode is the right use for us